### ISSUE TITLE
trailing comma

### ISSUE THREAD

**Issue by Brakkar** [palantir/tslint#1280]
### Bug Report

- __TSLint version__: `3.10.2`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__:  Node.js API 

#### TypeScript code being linted

```ts
 return {
            error: g.error.yes,
            model: undefined,
            errorBag: validationErrors
        }
```

#### Actual behavior

Reports an error of missing trailing comma after validationErrors
<img width="314" alt="capture d ecran 2016-05-27 a 15 29 48" src="https://cloud.githubusercontent.com/assets/5489200/15609420/3e46651c-2420-11e6-86b1-11781478af90.png">

#### Expected behavior

It should not. I can remove this behavior when moving the brace up, but I prefer it where it is.

**Comment by JKillian** [comment]
@Brakkar What settings are you using for the `trailing-comma` rile?

**Comment by adidahiya** [comment]
@Brakkar you can disable this check with `trailing-comma: { multiline: "never" }` (or omitting `multiline` altogether).

see http://eslint.org/docs/rules/comma-dangle for reference.

**Comment by Brakkar** [comment]
@adidahiya this did fix the issue. Thanks.
